submissions/Jemikalajah-Philip/Html-Css/Registration-form.html

```diff
@@ -19,8 +19,8 @@
                     </div>
```

Comment:
Please restructure your submission. HTML-CSS tasks in their folder and the JavaScript tasks in their own folder